### PR TITLE
feat: ami chart import script runner

### PR DESCRIPTION
**Pull request by YazeedLoonat**
This PR addresses https://github.com/bloom-housing/bloom/issues/4041

- [x] Addresses the issue in full

## Description
Creates a new script runner endpoint that takes in the standardized string and creates the new ami chart

## How Can This Be Tested/Reviewed?
a unit test was added, but you can test via localhost:3100/api login, and then leverage the new script runner endpoint

## Author Checklist:

- [ ] Added QA notes to the issue with applicable URLs
- [x] Reviewed in a desktop view
- [ ] Reviewed in a mobile view
- [ ] Reviewed considering accessibility
- [x] Added tests covering the changes
- [ ] Made corresponding changes to the documentation
- [x] Ran `yarn generate:client` and/or created a migration when required

## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates

### CODE REVIEW

**Review by ludtkemorgan — COMMENTED** [review]
I'm missing a bit of context about the script runner and I can get up to speed on that in a different environment. However, the linked ticket to this PR is about sorting the ami charts. This change doesn't appear to address that.

**Comment by YazeedLoonat** [comment]
Hey @ludtkemorgan oops! I linked the wrong ticket (switched a 1 and 4 at the end there) got that updated

I also sent you a message with a tech doc that lays out script runner as best as I could explain it!

**Review by ludtkemorgan — COMMENTED** [review]
I don't see any issues with the code and it worked for me locally. I just have a question on if we could consolidate this into one step

**Comment by YazeedLoonat** [comment]
Hey @ludtkemorgan nice catch! I got this updated!

**Review by mcgarrye — APPROVED** [review]
Looks great! if you can update the PR description to match the changed functionality that would be lovely. LGTM

File: api/src/services/script-runner.service.ts [hunk 1]
```diff
@@ -134,6 +138,58 @@ export class ScriptRunnerService {
     return { success: true };
   }
 
+  /**
+   *
+   * @param amiChartImportDTO this is a string it takes a very specific formatted like:
```
Comment (emilyjablonski) [inline api/src/services/script-runner.service.ts:143]: `very specific format`

File: api/src/services/script-runner.service.ts [hunk 2]
```diff
@@ -134,6 +138,58 @@ export class ScriptRunnerService {
...
+   * percentOfAmiValue_1 householdSize_1_income_value householdSize_2_income_value \n percentOfAmiValue_2 householdSize_1_income_value householdSize_2_income_value
+   * @returns a stringified version of AmiChartCreate DTO
+   * @description transfers data from foreign data into the database this api normally connects to. From this you can use it
```
Comment (emilyjablonski) [inline api/src/services/script-runner.service.ts:146]: Last sentence seems like it's maybe unfinished?

File: api/src/services/script-runner.service.ts (hunk 1 above)
Comment (YazeedLoonat) [reply api/src/services/script-runner.service.ts]: nice catch!

File: api/src/services/script-runner.service.ts (hunk 2 above)
Comment (YazeedLoonat) [reply api/src/services/script-runner.service.ts]: oop! I updated a couple things about the jsdocs!

**Review by emilyjablonski — APPROVED** [review]

**Pull request merged**